lms/djangoapps/experiments/utils.py

```diff
@@ -267,7 +284,46 @@ def get_experiment_user_metadata_context(course, user):
     """
     Return a context dictionary with the keys used by the user_metadata.html.
     """
-    # TODO: call get_base_experiment_metadata_context(), and then add in the bits that are only needed by user_metadata
+    if DEPRECATED_METADATA.is_enabled():
+        return get_deprecated_experiment_user_metadata_context(course, user)
```

Comment:
You could use https://github.com/joealcorn/laboratory to validate that the old way and the new way get you the same data, if you're expecting it to be identical.

Reply:
Done. See https://github.com/edx/edx-platform/pull/20544